src/App.js

```diff
@@ -42,6 +43,7 @@ class App extends Component {
             minimizeSubFormulas: true,
             truthTable: false,
             parenthesize: false,
+            lowenheim: false,
```

Comment:
I would like if `lowenheim` was renamed to `method` (or similar) and then we used the strings `"sve"` and `"lowenheim" to distinguish. This is more extensible if we want to play with more variants in the future.

Reply:
There is now a `unifyMethod` prop instead of `lowenheim`, and an enumeration of applicable methods from the Boolean unification module. Furthermore, a `select` input is used to change the method during runtime instead of a checkbox. Default method is `SVE`.